content/guides/getting-started/component-framework-configuration.md

```diff
@@ -258,89 +206,83 @@ export default defineConfig({
 You can find an example React project that uses Vite
 [here](https://github.com/cypress-io/cypress-component-examples/tree/main/setup-vite-react-app).
 
-<!-- Couldn't simply call this section "Vue" because using "## Vue" by itself killed the tabs in the code examples -->
+### Webpack
 
-## Vue 2 & Vue 3
-
-### Vue CLI
-
-For [Vue CLI](https://cli.vuejs.org/) projects, you will need to install the
-Cypress Vue adapter and the Cypress Webpack Dev Server as dev dependencies.
-
-The version of the Cypress Vue adapter you need depends on the version of Vue
-you are using.
-
-For Vue 3, install the latest `@cypress/vue` package:
+To configure component testing for a React app that uses a custom
+[Webpack](https://webpack.js.org/) config, you will need to configure a
+`devServer` with a framework of `react` and a bundler of `webpack` like so:
 
-<npm-or-yarn>
-<template #npm>
-
-```sh
-npm install --save-dev @cypress/vue @cypress/webpack-dev-server
-```
+<cypress-config-file>
+<template #js>
 
-</template>
-<template #yarn>
+```js
+const { defineConfig } = require('cypress')
 
-```sh
-yarn add -D @cypress/vue @cypress/webpack-dev-server
+module.exports = defineConfig({
+  component: {
+    devServer: {
+      framework: 'react',
+      bundler: 'webpack',
+      // optionally pass in webpack config
+      webpackConfig: require('webpack.config'),
+    },
+  },
+})
 ```
 
 </template>
-</npm-or-yarn>
-
-For Vue 2, install the `@cypress/vue@2` package:
-
-<npm-or-yarn>
-<template #npm>
-
-```sh
-npm install --save-dev @cypress/vue@2 @cypress/webpack-dev-server
-```
+<template #ts>
 
-</template>
-<template #yarn>
+```ts
+import { defineConfig } from 'cypress'
+import webpackConfig from 'webpack.config'
 
-```sh
-yarn add -D @cypress/vue@2 @cypress/webpack-dev-server
+export default defineConfig({
+  component: {
+    devServer: {
+      framework: 'react',
+      bundler: 'webpack',
+      // optionally pass in webpack config
+      webpackConfig,
+    },
+  },
+})
 ```
 
 </template>
-</npm-or-yarn>
+</cypress-config-file>
 
-<Alert type="info">
+If you don't provide one, Cypress will try to infer your webpack config. If
+Cypress cannot or you want to make modifications to your config, you can pass it
+in manually via the `webpackConfig` option.
```

Comment:
```suggestion
If you don't provide a webpack config, Cypress will try to infer it. If Cypress
cannot do so, or you want to make modifications to your config, you can specify
it via the `webpackConfig` option.
```

When you say "make modifications" what do you mean? Is the `webpackConfig` option merged in, or does it override?